### PR TITLE
Add populated tag to empty tables

### PR DESCRIPTION
**Pull request by dmohanty-r7**
MSP-13320

# Changelog
* Incompatible Changes
  * A "populated" tag is added when empty tables are loaded (in addition to non-empty tables that were already taged)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,16 +19,16 @@
     "grunt": "~0.4.5",
     "grunt-cli": "0.1.13",
     "grunt-contrib-clean": "^0.6.0",
-    "grunt-contrib-coffee": "~0.11.0",
+    "grunt-contrib-coffee": "~0.13.0",
```
Comment (limhoff-r7) [inline package.json]: Did you mean for the changing of tags to also update your dependencies?  This should really be separate no?

Comment (dmohanty-r7) [inline package.json]: I would assume we want the dependencies on the repo to be consistent with the package.json, right?

Comment (farias-r7) [inline package.json]: @dmaloney-r7 I think he meant they should be two PR's and two separate tickets.

Comment (KronicDeth) [inline package.json]: Yes, dependency updates, in general, should be separate PRs so reviewers know that you made a code change to be compatible with the updated dependencies.

**Comment by jvazquez-r7** [comment]
jenkins test this please
